README.md

```diff
@@ -422,15 +420,15 @@ end
 def show_employee_list(employees)
   employees.each do |employee|
     data = {
-      expected_salary: employee.calculate_expected_salary(),
-      experience: employee.get_experience()
+      expected_salary: employee.expected_salary,
+      experience: employee.experience
     }
 
     case employee.type
     when 'manager'
-      data.portfolio = employee.get_mba_projects()
+      data.portfolio = employee.mba_projects
     when 'developer'
-      data.github_link = employee.get_github_link()
+      data.github_link = employee.github_link
```

Comment:
These should be `data[:portfolio]` and `data[:github_link]` as data is a hash.

Reply:
Thanks! I've corrected it :)